lib/src/test/kotlin/integration/config-template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "rpc_url": "http://example-node-url.com/rpc",
+  "gateway_url": "http://example.com/gateway",
+  "feeder_gateway_url": "http://example.com/feeder_gateway",
+  "private_key": "0x123456789",
+  "integration_account_address": "0x123456789123456789123456789123456789123456789123456789123456789"
+}
```

Comment:
Let's see what other think, for me it is more intuitive to have it as a file and replace name and values, but maybe I'm the only one.

Reply:
I think this is better approach: https://github.com/software-mansion/starknet-jvm/pull/281#discussion_r1290173031

that way, you still have a file, I'm happy because it is not just some random json, and we get to make things a lot simpler :+1: